### DISCUSSION TITLE
Unable to install Foundry

### DISCUSSION THREAD

**Discussion by Kanishka227** [Cyfrin/foundry-full-course-cu#1173] [Q&A] [answered]
curl -L https://foundry.paradigm.xyz | bash 
```
This link is not working as i am getting connection timed out while trying to install. Can anyone help

**Comment by xyz899** [comment]
Hey !,

in order to solve your issue, check if your connection is correctly working since a timeout could be the result of a Poor Connection. If you have a vpn, it could be conflicting with the server call so try turn it off or change to a different location. 
Check if `curl` and especially `ssh` is up to date and correctly installed and be sure that :

1.  You are on MacOS or a one of the Linux Distros and both are updated to their last versions.
2. You are on Windows and you have installed WSL2 (Windows SubSystem for Linux) with the last update

If you didn't install here is a link explaining how by [Microsoft](https://learn.microsoft.com/en-us/windows/wsl/install#install-wsl-command)

**Reply by Kanishka227** [reply, marked as answer]
Previously i wasn't using vpn but after installing it everything worked. Thanks .
